ipaserver/install/server/install.py

```diff
@@ -855,7 +854,10 @@ def install(installer):
     service.print_msg("Applying LDAP updates")
     ds.apply_updates()
 
-    # Restart krb after configurations have been changed
+    # Restart ds and krb after configurations have been changed
+    service.print_msg("Restarting the directory server")
+    ds.restart()
```

Comment:
Looking at it I think it should probably be a krb.restart(), but I did not have any issue installing, so perhaps it can simply go now, I'll wait for @abbra to comment if he recalls anything around this part.